src/test/scala/strawman/collection/test/TraverseTest.scala

```diff
@@ -69,9 +69,8 @@ class TraverseTest {
 
   @Test
   def eitherSequenceTest: Unit = {
-    val xs3 = mutable.ListBuffer(Right("foo"), Left(0), Right("bar"))
+    val xs3: mutable.ListBuffer[Either[Int, String]] = mutable.ListBuffer(Right("foo"), Left(0), Right("bar"))
```

Comment:
> If this is too greedy, one could restrict it to only operate on enums

You wouldn't have this problem with enums because the return types of their apply methods are always widened to the enum type